Tests/scs-compatible-iaas.yaml

```
@@ -152,7 +152,7 @@ modules:
     testcases:
       - id: standard-images-check
         tags: [mandatory]
-        description: |
+        description: >
           The environment fulfills all requirements of https://docs.scs.community/standards/scs-0104-v1-standard-images
```

```suggestion
          The environment should fulfill all requirements of https://docs.scs.community/standards/scs-0104-v1-standard-images
```
Maybe the wording with "should fulfill" is better. As it is unknown and depends on the result of the check.
This also avoid any perceived contradiction between `FAILED` and `fulfills`.

---

Or maybe "Passing indicates that the environment fulfills all requirements ..."

---

```suggestion
          Must fulfill all requirements of https://docs.scs.community/standards/scs-0104-v1-standard-images
```

---

I like this one the best thus far.